pyap/address.py

```diff
@@ -32,6 +32,7 @@ class Address:
     street_type: Union[str, None] = None
     building_id: Union[str, None] = None
     postal_code: Union[str, None] = None
+    single_street_name: Union[str, None] = None
```

Comment:
why do we need this? This feels like we would need to check 2 things everytime

Reply:
@DeviousStoat We have two ways to match a street: either it's a `street_name` + `street_type` or a `single_street_name`, so I think it's only natural that we include the latter field to the Address model. We still have the `full_street` field which is the end product that we mostly use.